x/data/server/testsuite/suite.go

```diff
@@ -41,106 +42,168 @@ func (s *IntegrationTestSuite) TearDownSuite() {
+	}
+	contentHash := &data.ContentHash{Sum: &data.ContentHash_Graph_{Graph: graphHash}}
+
+	//// anchor some data
+	anchorRes, err := s.msgClient.AnchorData(s.ctx, &data.MsgAnchorDataRequest{
+		Sender: s.addr1.String(),
+		Hash:   contentHash,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(anchorRes)
+
+	// anchoring same data twice is a no-op
+	_, err = s.msgClient.AnchorData(s.ctx, &data.MsgAnchorDataRequest{
+		Sender: s.addr1.String(),
+		Hash:   contentHash,
+	})
+	s.Require().NoError(err)
+
+	// can query data and get timestamp
+	queryRes, err := s.queryClient.ByHash(s.ctx, &data.QueryByHashRequest{
+		Hash: contentHash,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(queryRes)
+	s.Require().NotNil(queryRes.Entry)
+	ts := queryRes.Entry.Timestamp
+	s.Require().NotNil(ts)
+	s.Require().Empty(queryRes.Entry.Signers)
+	s.Require().Empty(queryRes.Entry.Content)
+	iri, err := graphHash.ToIRI()
+	s.Require().NoError(err)
+	s.Require().Equal(iri, queryRes.Entry.Iri)
+
+	// can sign data
+	_, err = s.msgClient.SignData(s.ctx, &data.MsgSignDataRequest{
+		Signers: []string{s.addr1.String()},
+		Hash:    graphHash,
+	})
+	s.Require().NoError(err)
+
+	// can retrieve signature, same timestamp
+	// can query data and get timestamp
+	queryRes, err = s.queryClient.ByHash(s.ctx, &data.QueryByHashRequest{Hash: contentHash})
+	s.Require().NoError(err)
+	s.Require().NotNil(queryRes)
+	s.Require().Equal(ts, queryRes.Entry.Timestamp)
+	s.Require().Len(queryRes.Entry.Signers, 1)
+	s.Require().Equal(s.addr1.String(), queryRes.Entry.Signers[0].Signer)
+	s.Require().Empty(queryRes.Entry.Content)
+
+	// query data by signer
+	bySignerRes, err := s.queryClient.BySigner(s.ctx, &data.QueryBySignerRequest{
+		Signer: s.addr1.String(),
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(bySignerRes)
+	s.Require().Len(bySignerRes.Entries, 1)
+	s.Require().Equal(queryRes.Entry, bySignerRes.Entries[0])
+
+	// another signer can sign
+	_, err = s.msgClient.SignData(s.ctx, &data.MsgSignDataRequest{
+		Signers: []string{s.addr2.String()},
+		Hash:    graphHash,
+	})
+	s.Require().NoError(err)
+
+	// query data by signer
+	bySignerRes, err = s.queryClient.BySigner(s.ctx, &data.QueryBySignerRequest{
+		Signer: s.addr2.String(),
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(bySignerRes)
+	s.Require().Len(bySignerRes.Entries, 1)
+	s.Require().Equal(contentHash, bySignerRes.Entries[0].Hash)
+
+	// query and get both signatures
+	queryRes, err = s.queryClient.ByHash(s.ctx, &data.QueryByHashRequest{Hash: contentHash})
+	s.Require().NoError(err)
+	s.Require().NotNil(queryRes)
+	s.Require().Equal(ts, queryRes.Entry.Timestamp)
+	s.Require().Len(queryRes.Entry.Signers, 2)
+	signers := make([]string, len(queryRes.Entry.Signers))
+	for _, signer := range queryRes.Entry.Signers {
+		signers = append(signers, signer.Signer)
+	}
+	s.Require().Contains(signers, s.addr1.String())
+	s.Require().Contains(signers, s.addr2.String())
+	s.Require().Empty(queryRes.Entry.Content)
```

Comment:
why we don't declare 
```
require := s.Require()
```

at the beginning of the function - it would make the code easier to read.